src/SFA.DAS.TrackProgress.Database/Tables/ApprenticeProgress.sql

```diff
@@ -5,7 +5,9 @@
     [Uln] bigint NOT NULL,
     [StartDate] date NOT NULL,
     [ApprovalId] bigint NOT NULL,
-    [OnTrack] bit NOT NULL,
+    [ApprovalContinuationId] bigint NULL,
+    [ProgressDataVersion] bigint NOT NULL,
```

Comment:
where does this value get populated from. BTW I think it's a good idea

Reply:
Currently `ProgressDataVersion` is always 1 as we only have the single version of the progress JSON.  We will have to deserialise the JSON based on this number as soon as we introduce a change to the JSON schema.